src/main/java/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -117,11 +117,10 @@ private void pushFightLoopOnStack(final boolean firstRun) {
   }
 
   private boolean shouldFightAirBattle() {
-    if (m_isBombingRun) {
-      return Match.anyMatch(m_attackingUnits, Matches.unitIsStrategicBomber()) && !m_defendingUnits.isEmpty();
-    } else {
-      return !m_attackingUnits.isEmpty() && !m_defendingUnits.isEmpty();
-    }
+    return ((m_isBombingRun && m_attackingUnits.stream().anyMatch(Matches.unitIsStrategicBomber()))
+        || !m_attackingUnits.isEmpty())
+        && !m_defendingUnits.isEmpty();
```

Comment:
This is a bit hard to read.

Reply:
I changed the order to be able to see the encapsulation better as part of the rebase, should be better now